appartment/forms/auth_forms.py

```diff
@@ -0,0 +1,62 @@
+# forms.py
+from django import forms
+from ..models import User
+
+
+class LoginForm(forms.Form):
+    """
+    Form đăng nhập người dùng
+    """
+
+    email = forms.EmailField(
+        max_length=255,
+        widget=forms.EmailInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Nhập email của bạn",
+                "autofocus": True,
+                "required": True,
+            }
+        ),
+        label="Email",
+    )
+
+    password = forms.CharField(
+        widget=forms.PasswordInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Nhập mật khẩu",
+                "required": True,
+            }
+        ),
+        label="Mật khẩu",
+    )
+
+    remember_me = forms.BooleanField(
+        required=False,
+        widget=forms.CheckboxInput(attrs={"class": "form-check-input"}),
+        label="Ghi nhớ đăng nhập",
+    )
+
+    def clean(self):
+        """
+        Validate form data
+        """
+        cleaned_data = super().clean()
+        email = cleaned_data.get("email")
+        password = cleaned_data.get("password")
+
+        if email and password:
+            # Kiểm tra xem user có tồn tại không
+            try:
+                user = User.objects.get(email=email)
```

Comment:
Chỗ này mình có thể dùng filter rồi lấy first object, sẽ ko raise exception not exist
bên dưới thêm xử lý `if not user` là đc 
```suggestion
                user = User.objects.filter(email=email).first()
```